pkg/catalog/interrogate.go

```diff
@@ -356,3 +356,13 @@ func GetApp(
 	cr.AppSpec = appCR
 	return appCR, nil
 }
+
+// GetSvcClusterDomainBase fetches the cluster domain base (for (e.g) .cluster.local) from
+// config CR
+func GetSvcClusterDomainBase() string {
+	kdConfigCR, err := observer.GetKDConfig(shared.KubeDirectorGlobalConfig)
+	if err != nil {
+		return ""
```

Comment:
may be should just return the default here?

Reply:
We have our config object cached in memory; see pkg/shared/globalconfig.go.